.github/workflows/run_renovate.yml

```diff
@@ -0,0 +1,20 @@
+name: Renovate-Dry-Run
```

Comment:
this file needs to go

Reply:
I kept a chaned file in the repo. Should be nice-to have, if we ever need to update the renovate.json